CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Nix
+
+A big welcome and thank you for considering contributing to Nix! It's people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+It also communicates that you agree to respect the time of the developers managing and developing these open source projects.
+In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Code of conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication.
+By participating and contributing to this project, you agree to uphold our [Code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/).
+
+## Report a bug or suggest a feature
```

Comment:
Reading the [RoR guide on feature requests](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#what-about-feature-requests-questionmark), I think we should not actively encourage feature requests, but also not be as blunt and just close them away.

In my opinion, feature requests are somewhat a distraction, and while we need but haven't yet found a structured and predictably productive way to discuss them, there's also no need to further promote that ad hoc practice.

I suggest just not talking about feature suggestions at all, feels like the right middle ground until further notice.

```suggestion
## Report a bug
```

Would need fixups in list items following this